docs/upgradetest.md

```diff
@@ -0,0 +1,42 @@
+This guide is for Harbor upgrade test  
+=======
+
+# Before upgrade
+
+## Prepare data  
+1. At least three users, at least one has admin role.  
+2. Create at least one private and one public project.  
+3. Add several members to project, each role need one.  
+4. Login harbor as project admin, push at least 2 images into project, include signed. pull each image at least one time. 
+5. Login harbor as project developer, push at least 2 images into project, include signed. pull each image at least one time. 
+6. Repeate 4 and 5 to all projects.  
```

Comment:
For step 4~6,  it's still too confusing.
I think for verification we only need to sign one image.
And it would be helpful if you can name the project, user, and explicitly document what images should be pushed, signed and scanned.